integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -140,12 +146,12 @@ export class CdcJsService implements OnDestroy {
    * @param baseSite
    */
   protected addCdcEventHandlers(baseSite: string): void {
-    (this.winRef.nativeWindow as { [key: string]: any })?.[
+    this.gigyaSDK = (this.winRef.nativeWindow as { [key: string]: any })?.[
       'gigya'
-    ]?.accounts?.addEventHandlers({
-      onLogin: (...params: any[]) => {
-        this.zone.run(() => this.onLoginEventHandler(baseSite, ...params));
-      },
+    ];
+    this.gigyaSDK?.accounts?.addEventHandlers({
+      onLogin: (...params: any[]) =>
+        this.onLoginEventHandler(baseSite, ...params),
```

Comment:
To fix broken change detection after user login, let's wrap it withing ngZone
```suggestion
        this.zone.run(() => this.onLoginEventHandler(baseSite, ...params)),
```

Reply:
noted. thanks @Matejk00  and @Platonn.

@armanozak FYI we have to bring back the `NgZone` due to context switch outside Angular, the UI seems unresponsive.